CONTRIBUTING.md

```diff
@@ -4,26 +4,85 @@ Vald is an open source project.
 
 We appreciate your help!
 
-## Before filing an issue
+## Table of Contents
 
-If you are unsure whether you have found a bug, please consider asking in the [Vald Slack](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) first. If the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.
+- [Contributing Issue](#Contributing-Issue)
+  - [Before filing an issue](#Before-filing-an-issue)
+  - [Filing issues](#Filing-issues)
+  - [Bug Report](#Bug-Report)
+  - [Proposal](#Proposal)
+  - [Feature Request](#Feature-Request)
+  - [Security Issue Report](#Security-Issue-Report)
+- [Contributing Code](#Contributing-Code)
+  - [Before contributing code](#Before-contributing-code)
+  - [Contributing code](#Contributing-code)
+  - [Branch naming convention](#Branch-naming-convention)
 
-## Filing issues
+## Contributing Issue
 
-Sensitive security-related issues should be reported to the security channel in the [Vald Slack](https://join.slack.com/t/vald-community/shared_invite/zt-db2ky9o4-R_9p2sVp8xRwztVa8gfnPA) .
+### Before filing an issue
 
-Otherwise, when filing an issue, make sure to answer these five questions:
+If the behavior you are seeing is confirmed as a bug or issue, it can easily be re-raised in the issue tracker.
```

Comment:
Can you tell me what is `issue tracker`?

Reply:
For example, this link describes the `issue tracker`.
https://github.com/xamarin/xamarin-android/wiki/Submitting-Bugs,-Feature-Requests,-and-Pull-Requests#using-the-issue-tracker

Reply:
Not sure if it is what you mean... 
```suggestion
If you can determine the problem you are facing is a bug or issue, you can easily submit in the issue tracker.
```

Reply:
:memo:

issue tracker is issues
https://github.com/vdaas/vald/issues